RSSParser/RSSParser.m

```diff
@@ -199,6 +211,27 @@ -(NSURL *)getNSURLFromString:(NSString *)string {
     }
 }
 
+-(NSDate *)getDateFromString:(NSString *)string {
+    NSDate *date = nil;
+    if (string) {
+        date = [_formatter dateFromString:string];
+        if (!date) {
+            date = [_ISO8601formatter dateFromString:string];
+        }
+        if (!date) {
+            NSRange range = [string rangeOfString:@" " options:NSBackwardsSearch];
+            NSString *threeLetterZone = [string substringFromIndex:range.location+1];
+            NSTimeZone *timeZone = [NSTimeZone timeZoneWithAbbreviation:threeLetterZone];
+            if (timeZone)
+            {
+                NSString *gmtTime = [string stringByReplacingOccurrencesOfString:threeLetterZone withString:@"GMT"];
```

Comment:
oh, also there is only `secondsFromGMT` on a `TimeZone` object. There is no such thing as `secondsFromEST`